Fails to compile with glibc 2.29/gcc 8+: src/os/unix/pws_time.h:17:18: error: conflicting declaration ‘typedef uint64_t __time_t’

Compiling with glibc version 2.29, gcc 8.2.0. Log of build failure attached.

[build-failure.log](https://github.com/pwsafe/pwsafe/files/2994587/build-failure.log)

@ronys Apologies, it seems I forgot to capture stderr when I was building. I double checked the attached build log includes it this time. I also pasted (one of) the relevant errors below. This is on Solus, using their container-like `solbuild` build environment.

```
In file included from /usr/include/stdint.h:27,
                 from /usr/lib64/gcc/x86_64-solus-linux/8/include/stdint.h:9,
                 from /usr/include/c++/8/cstdint:41,
                 from /usr/include/c++/8/bits/char_traits.h:503,
                 from /usr/include/c++/8/string:40,
                 from /home/build/YPKG/root/pwsafe/build/pwsafe-1.07BETA/src/os/typedefs.h:20,
                 from /home/build/YPKG/root/pwsafe/build/pwsafe-1.07BETA/src/core/AES.h:16,
                 from /home/build/YPKG/root/pwsafe/build/pwsafe-1.07BETA/src/core/AES.cpp:13:
/home/build/YPKG/root/pwsafe/build/pwsafe-1.07BETA/src/os/unix/pws_time.h:17:18: error: conflicting declaration ‘typedef uint64_t __time_t’
 typedef uint64_t __time64_t;
                  ^~~~~~~~~~
/usr/include/bits/types.h:160:26: note: previous declaration as ‘typedef long int __time_t’
 __STD_TYPE __TIME_T_TYPE __time_t; /* Seconds since the Epoch.  */
                          ^~~~~~~~
```

[build-failure-with-stderr.log](https://github.com/pwsafe/pwsafe/files/3011023/build-failure-with-stderr.log)

--- a/src/os/unix/pws_time.h
+++ b/src/os/unix/pws_time.h
@@ -13,8 +13,11 @@
 #ifdef __FreeBSD__
 #include <time.h>
 #endif
+
 typedef time_t __time32_t;
+#ifndef __time64_t 
 typedef uint64_t __time64_t;
+#endif
 
 extern int localtime64_r(const __time64_t *timep, struct tm *result);
 

